NEWS

```diff
@@ -5,6 +5,9 @@ Current sponsors include github users: brentfrow, brazeon, and q66 (Daniel Koles
 Chimera Linux). Development of Dinit has also received financial support from Artix Linux.
 I am grateful for all the support, both financial and otherwise, that has been received.
 
+NOTE: This release has some minor backwards-incompatible changes about logfiles. Please read
```

Comment:
This note isn't needed. There are several backwards-incompatible changes, not all of them to do with logfiles, and they are already listed in the changes section. (I will revise these notes before release. You don't need to add changes to the NEWS as part of a pull request).